src/plugins/data/public/search/aggs/test_helpers/mock_data_services.ts

```diff
@@ -41,11 +40,10 @@ import {
 export function mockDataServices() {
```

Comment:
This still lives as a test helper inside `search.aggs`, but might be more generically useful elsewhere in the data plugin. Perhaps we consider eventually moving it up to a different directory